src/views/Home/RecentActivity/styles.scss

```diff
@@ -0,0 +1,52 @@
+.recent-activity {
+    display: flex;
+    flex-direction: column;
+
+    .header {
+        padding: var(--dui-spacing-small) 0;
+    }
+
+    .content-container {
+        display: flex;
+        flex-direction: column;
+        border-radius: 10px;
```

Comment:
If Card component is used this is unnecessary.

Reply:
The Card component does not have any styling related to border-radius. When not used, the box is just a rectangle with pointed corners.